AppCenterAuth/AppCenterAuthTests/MSAuthTests.m

```diff
@@ -1216,4 +1220,67 @@ - (void)testRefreshNeededWithNilAccountTriggersAnonymous {
   [authTokenContextMock stopMocking];
 }
 
+- (void)testContinuePendingSigninAfterConfigDownload {
+  
+  // If
+  [[MSAuth sharedInstance] setEnabled:YES];
+  id authMock = OCMPartialMock(self.sut);
+  MSSignInCompletionHandler handler = ^(MSUserInformation *_Nullable userInformation, NSError *_Nullable error) {
+    self.signInUserInformation = userInformation;
+    self.signInError = error;
+  };
+  
+  // When
+  OCMStub([authMock sharedInstance]).andReturn(authMock);
```

Comment:
lines 1234-1236 should be in `// If` block

Reply:
@dtrabo